relbench/tasks/stackex.py

```diff
@@ -172,7 +164,7 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
             LEFT JOIN badges b
                 ON u.Id = b.UserID
                 AND b.Date > t.timestamp
-                AND b.Date <= t.timestamp + INTERVAL 2 years
+                AND b.Date <= t.timestamp + INTERVAL {self.timedelta}
```

Comment:
@joshr17 I believe setting this to 2 years while using timedelta=1 year will result in label information leakage into the validation set. In general, we shouldn't generate labels beyond `timedelta`.

Reply:
Good spot!